server/controllers/class/putAssignmentTeacher.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response, NextFunction } from 'express';
+import { updateGradPayloadValidate } from '../../utils/validation';
+import { putAssignmentTeacherQuery } from '../../queries/class';
+
+const putAssignmentTeacher = async (req:Request, res:Response, next:NextFunction) => {
+  try {
+    const { assignmentId } = req.params;
+    const { grade, studentId } = req.body;
+
+    await updateGradPayloadValidate({ grade, studentId });
+
+    const [, data]:any = await putAssignmentTeacherQuery(assignmentId, grade, studentId);
+    res.status(201).json({ data, msg: 'updating successfully' });
+  } catch (error) {
+    next(error);
+  }
```

Comment:
The error thrown by your validation method is not customized, so check it by its name and customized it.